### ISSUE TITLE
Newly created meteor project doesn't run

### ISSUE THREAD

**Issue by gischer** [meteor/meteor#8548]
Meteor reports itself as version 1.4.3.2.  Running on Mac OSX Sierra 10.12

running 

```
  > meteor create meteor-create-test
  > cd meteor-create-test
  > meteor

```
gives the following error:

```
[[[[[ ~/SweetvineSystems/meteor-create-test ]]]]]

=> Started proxy.                             
=> Started MongoDB.                           
W20170330-15:01:46.252(-7)? (STDERR) /Users/gischer/.meteor/packages/meteor-tool/.1.4.3_2.1u603c2++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20170330-15:01:46.354(-7)? (STDERR) 						throw(ex);
W20170330-15:01:46.355(-7)? (STDERR) 						^
W20170330-15:01:46.355(-7)? (STDERR) 
W20170330-15:01:46.355(-7)? (STDERR) TypeError: Cannot read property 'env' of undefined
W20170330-15:01:46.355(-7)? (STDERR)     at meteorInstall.node_modules.meteor.ddp-server.server_convenience.js (packages/ddp-server/server_convenience.js:1:5)
W20170330-15:01:46.356(-7)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:205:9)
W20170330-15:01:46.356(-7)? (STDERR)     at require (packages/modules-runtime.js:128:16)
W20170330-15:01:46.357(-7)? (STDERR)     at /Users/gischer/SweetvineSystems/meteor-create-test/.meteor/local/build/programs/server/packages/ddp-server.js:2158:1
W20170330-15:01:46.357(-7)? (STDERR)     at /Users/gischer/SweetvineSystems/meteor-create-test/.meteor/local/build/programs/server/packages/ddp-server.js:2169:3
W20170330-15:01:46.357(-7)? (STDERR)     at /Users/gischer/SweetvineSystems/meteor-create-test/.meteor/local/build/programs/server/boot.js:303:34
W20170330-15:01:46.358(-7)? (STDERR)     at Array.forEach (native)
W20170330-15:01:46.358(-7)? (STDERR)     at Function._.each._.forEach (/Users/gischer/.meteor/packages/meteor-tool/.1.4.3_2.1u603c2++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20170330-15:01:46.358(-7)? (STDERR)     at /Users/gischer/SweetvineSystems/meteor-create-test/.meteor/local/build/programs/server/boot.js:128:5
W20170330-15:01:46.359(-7)? (STDERR)     at /Users/gischer/SweetvineSystems/meteor-create-test/.meteor/local/build/programs/server/boot.js:352:5
=> Exited with code: 1
```
`

I have other meteor projects (running older versions of meteor) which run correctly from the same shell.

**Comment by abernix** [comment]
Hmm.  This certainly works for me with the same versions.

Looks like this is stemming from [`ddp-server/server_convenience.js`](https://github.com/meteor/meteor/blob/devel/packages/ddp-server/server_convenience.js#L11).  This suggests something is quite wrong as the only `env` references there are done on the [`process`](https://nodejs.org/dist/latest-v4.x/docs/api/process.html) object which should always exist on the server.

Unfortunately, I might suggest doing a fresh installation of Meteor:

    curl https://install.meteor.com/ | sh

**Comment by gischer** [comment]
Thank you.  This worked, rebooting didn't.
Here's what happened:
I rebooted and created a test project, then attempted to run it.  It failed, in the same way.

I then reinstalled meteor per your instruction.  I then attempted to run the project that I had previously created.  It downloaded a couple of things (they went by too quick to see what), and then it ran correctly.

This strikes me as a bit odd, but it's working now, and I'm fine if you want to close this as resolved.
